### PR TITLE
Admin crud tests

### LINKED ISSUE #9: Test updated user service that creates users and renders user lists

### LINKED ISSUE #16: Test Updated User and Tuit services

### PR DESCRIPTION
**Pull request by aliu19**

### CODE REVIEW

File: src/tests/users-service.test.js [hunk 1]
```diff
@@ -197,3 +197,89 @@ describe('user can update their information', () => {
...
+  })
+
+  test('user can search by username', async () => {
+    const users = await searchByUsername('and');
+
+    expect(users.length).toEqual(2);
+  });
+})
+
+describe('user can search by username', () => {
```
Comment (caijiah) [inline src/tests/users-service.test.js:245]: This test runs correctly, but are these two the same tests?

File: src/tests/tuits-service.test.js [hunk 2]
```diff
@@ -205,4 +207,40 @@ describe('can retrieve all tuits with REST API', () => {
             expect(tuit.postedBy._id).toEqual(author._id)
         })
     })
-});
\ No newline at end of file
+});
+
+describe('can search tuit by tuit', () => {
```
Comment (caijiah) [inline src/tests/tuits-service.test.js:212]: This one fails, I can try if I can fix this.
Also, could we test updating a tuit?

Comment (aliu19) [reply src/tests/tuits-service.test.js:212]: Sure, but I didn't update that from the source code which is why I didn't write any additional tests for that.

Comment (caijiah) [reply src/tests/tuits-service.test.js:212]: I didn't write tests for updateTuit in our starting code, because it is not required in previous assignments.

**Pull request merged**